src/Form/QuestionType/QuestionTypeDateTime.php

```diff
@@ -0,0 +1,374 @@
+                        let inputPlaceholder = {{ placeholders.input.date|json_encode|raw }};
+                        let defaultValuePlaceholder = {{ placeholders.default_value.date|json_encode|raw }};
+                        if (isDateEnabled && isTimeEnabled) {
+                            inputType = 'datetime-local';
+                            inputPlaceholder = {{ placeholders.input['datetime-local']|json_encode|raw }};
+                            defaultValuePlaceholder = {{ placeholders.default_value['datetime-local']|json_encode|raw }};
+                        } else if (isTimeEnabled) {
+                            inputType = 'time';
+                            inputPlaceholder = {{ placeholders.input.time|json_encode|raw }};
+                            defaultValuePlaceholder = {{ placeholders.default_value.time|json_encode|raw }};
+                        }
+
+                        if (isDefaultValueCurrentTime.is(':checked')) {
+                            inputType = 'text';
+
+                            // Clear the value
+                            dateInput.val('');
+                        }
+
+                        dateInput.prop('type', inputType);
+                        dateInput.prop('placeholder', inputPlaceholder);
+                        isDefaultValueCurrentTime.siblings('span').text(defaultValuePlaceholder);
+                    }
+                </script>
+TWIG;
+        }
+
+        $twig = TemplateRenderer::getInstance();
+        return $twig->renderFromStringTemplate($template, [
+            'question'          => $question,
+            'default_value'     => $this->isDefaultValueCurrentTime($question)
+                ? '' : $question->fields['default_value'] ?? '',
+            'input_type'        => $this->getInputType($question),
+            'input_type_ignore_text'        => $this->getInputType($question, true),
+            'is_default_value_current_time' => $this->isDefaultValueCurrentTime($question),
+            'placeholders'      => $this->getPlaceholders()
+        ]);
+    }
+
+    #[Override]
+    public function renderAdministrationOptionsTemplate(?Question $question): string
+    {
+        $template = <<<TWIG
+            {% set rand = random() %}
+
+            <div class="d-flex gap-2">
+                <label class="form-check form-switch mb-0">
+                    <input type="hidden" name="is_date_enabled" value="0"
+                    data-glpi-form-editor-specific-question-extra-data>
+                    <input class="form-check-input" type="checkbox" name="is_date_enabled"
+                        id="is_date_enabled_{{ rand }}"
+                        value="1" {{ is_date_enabled ? 'checked' : '' }}
+                        onchange="handleDateAndTimeCheckbox_{{ rand }}(this)"
+                        data-glpi-form-editor-specific-question-extra-data>
+                    <span class="form-check-label">{{ labels.date|e('html') }}</span>
```

Comment:
`html` is the default strategy. No need to specify it.
Same remark for other occurences.

```suggestion
                    <span class="form-check-label">{{ labels.date }}</span>
```